server/client.go

```diff
@@ -199,14 +200,14 @@ type outbound struct {
 	nb  net.Buffers   // net.Buffers for writev IO
 	sz  int           // limit size per []byte, uses variable BufSize constants, start, min, max.
 	sws int           // Number of short writes, used for dynamic resizing.
-	pb  int64         // Total pending/queued bytes.
-	pm  int64         // Total pending/queued messages.
+	pb  int           // Total pending/queued bytes.
```

Comment:
From go doc: https://golang.org/src/builtin/builtin.go?s=2249:2261#L65, `int` is at least 32 bits but may be more. I think recalling that on amd64 it will actually be 64bit, so change should not change much. 
If you really want 32 bits, then we should be explicit with `int32`. Then question is, is that ok to reduce `pb` (and `mp`) to 2^31 (2GB)? I think so, but unless we fail user setting max pending to more than 2GB, we have to be careful of rollover (especially `mp`).

Reply:
ok so maybe good for us to specify exactly int32 and have some tests to check max configured values. Will add tomorrow.